src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_scaffold.scss

```diff
@@ -17,17 +18,212 @@ main, #pagehdr, footer {
+    }
+
+    @include flex;
+    @include flex-items-column;
+    @include flex-items-nowrap;
+
+    .article-info-block {
+      @include flex;
+      @include flex-items-row;
+      @include flex-items-nowrap;
+      @include flex-items-space-between;
+      @include breakpoint(sm) {
+        @include flex-items-space-around;
+      }
+
+      .date-doi {
+        @include breakpoint(sm) {
+          text-align: center;
+        }
+        font-family: $headings-font-family;
+        margin-bottom: 4ch;
+
+        .date {
+          font-size: $font-size-big;
+          color: $brand-main;
+        }
+        .doi {
+          font-size: $font-size-small;
+          a {
+            text-decoration: none;
+            color: $brand-accent;
+          }
+        }
+      }
+      .published {
+        @include breakpoint(sm) {
+          display: none;
+        }
+      }
+      .buttons {
+        @include breakpoint(sm) {
+          display: none;
+        }
+        @include flex;
+        @include flex-items-row;
+        @include flex-items-nowrap;
+        .share {
+          @include icon_label('share_icon.svg', 'Share');
+        }
+        .citations {
+          @include icon_label('download_citations.svg', 'Citations');
+        }
+      }
+    }
+
+    .article-content-block {
+      max-width: 720px;
+      align-self: center;
+
+      .article-title {
+        @include breakpoint(sm) {
+          text-align: center;
+        }
+      }
+
+      .article-post-title {
+        align-self: center;
+        display: inline-block;
+        @include breakpoint(sm) {
+          width: 100%;
+          text-align: center;
+        }
+
+        .fulltext {
+          @include icon_label('download_citations.svg', 'Full Text');
+
+        }
+      }
+
+      .article-body {
+        margin-top: 4ch;
+
+        h2.collapse-toggle {
+          border-top: solid 1px $color-border;
+          padding-top: 1ch;
+          padding-bottom: 1ch;
+          margin-bottom: 0;
+          font-weight: bold;
+        }
+        h2.collapse-toggle:after {
+          content: "+";
+          width: 20px; height: 20px; font-size: 20px; line-height: 20px; text-align: center;
+          float: right;
+          color: #ffffff;
+          background-color: $brand-main;
+          cursor: pointer;
+        }
+        h2.collapse-toggle.active {
+          height: 0px;
+        }
+        h2.collapse-toggle.active:after {
+          content: "\2715";
+          background-color: $brand-accent;
+          font-size: 12px;
+        }
+        h2.collapse-toggle.active > span {
+          max-width: 116px;
+          display: inline-block;
+          word-break: break-all;
+        }
+        .collapse-box {
+          position: relative;
+          width: calc(100% - 128px);
+          left: 128px;
+          top: -1ch;
+          z-index: -1;
+        }
+
+        @include breakpoint(sm) {
+          h2.collapse-toggle {
+            text-align: center;
```

Comment:
You can use indentation in .scss to read:

h2.collapse-toggle {
  &.active {
      &:active{

}
}
}